tests/utils/testing_utils.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import inspect
+
+
+def get_scenes_to_test(module_name):
+    """Get all Test classes of the module from which it is called. Used to fetch all the SceneTest of the module.
+
+    Parameters
+    ----------
+    module_name : :class:`str`
+        The name of the module tested.
+
+    Returns
+    -------
+    :class:`list`
```

Comment:
unspecific types so generic type; gotta specify type of list
```suggestion
    List[:class:`type`]
```